data/seeds/00-add-users.js

```diff
@@ -0,0 +1,16 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex("users")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("users").insert([
+        { username: "user1", password: "1234", department: "finance" },
+        { username: "user2", password: "1234", department: "finance" },
+        { username: "user3", password: "1234", department: "finance" },
+        { username: "user4", password: "1234", department: "finance" },
+        { username: "user5", password: "1234", department: "finance" },
+        { username: "paco", password: "1234", department: "accounts" }
```

Comment:
Do not create seed files for users table. use register endpoint to create new users